### ISSUE TITLE
Profres

### ISSUE THREAD

**Issue by Profres** [stascorp/rdpwrap#985]
RD Wrapper has stopped working with Windows 10 Build 18363.657. Is there an updated rdwrap.ini to deal with this?

**Comment by asmtron** [comment]
use autoupdate fort the new 18363.657
https://github.com/stascorp/rdpwrap/pull/859

**Comment by rebelliouswhiz** [comment]
> use autoupdate fort the new 18363.657
> #859

Hi asmtron @asmtron , I think OS version 18363.657 is not listed in the latest ini... The 18362.657 is listed.

https://github.com/asmtron/rdpwrap/blob/master/res/rdpwrap.ini

**Comment by affinityv** [comment]
"OS version" and "termserv.dll" version are different; that shouldn't be a problem.
RDPConf.exe reports the "termserv.dll" version, not the OS version....

Somebody correct me if I am wrong about that.

**Comment by rebelliouswhiz** [comment]
@affinityv I think you are right.

@Profres @asmtron I tested it again, and I found the update script didn't help me, and I had to download the rdpwrap.ini directly through the repository and replace the one in the folder.

However, it's not necessary to be the script's problem. I am using proxy software to re-direct my Http/Https links,  and they sometimes cause problems.

I am happy with the result. Thank you very much for your works @asmtron and thank you for your help @affinityv !
